asf_heat_pump_suitability/getters/load_boundaries.py

```diff
@@ -0,0 +1,51 @@
+import geopandas as gpd
+import shapely
+from typing import List
+
+from asf_heat_pump_suitability import config
+
+
+def load_gdf_local_authority_boundaries(
+    select_las: str | List[str] = None,
+) -> gpd.GeoDataFrame:
+    """
+    Load boundaries for specified Local Authority Districts. CRS British National Grid (27700).
+
+    Args:
+        select_las (str | List[str]): selected Local Authorities to load boundaries for. Optional. Default None to load all UK
+        Local Authority boundaries.
+
+    Returns:
+        gpd.GeoDataFrame: boundaries for specified Local Authority Districts
+    """
+    la_boundaries_gdf = gpd.read_file(
+        config["data"]["geodata"]["boundaries"]["UK_ons_lad_bounds"]
+    )
+
+    if not select_las:
+        print("Loading Local Authority boundaries for UK...")
+        return la_boundaries_gdf
+    elif isinstance(select_las, str):
+        print(f"Loading Local Authority boundaries for {select_las}...")
+        return la_boundaries_gdf[
+            la_boundaries_gdf["LAD23NM"].str.contains(select_las.title())
+        ]
```

Comment:
```suggestion
        la_boundaries_gdf = la_boundaries_gdf[
            la_boundaries_gdf["LAD23NM"].str.contains(select_las.title())
        ]

        # Raise exception if boundaries are not found for LA specified by select_las
        if len(la_boundaries_gdf) == 0:
            raise Exception(
                f"Could not find boundaries for the following Local Authorities: {select_las}"
            )
        else:
            return la_boundaries_gdf
```

In the same way it's done for no matches in a list of LAs, I suggest also raising an exception if no match is found when just a single LA string is specified

Reply:
Good idea :)